docs/Installation.md

```diff
@@ -23,74 +24,69 @@ Since we use the Symfony framework, the [minimal requirements of Symfony][2] app
 
 Apart from these you will also need the following :
 
-- [Node.js][3]
-- [Sass][4]
-- [Bower][5]
-- [Grunt][6]
+- [Node.js][4]
+- [Sass][5]
+- [Bower][6]
+- [Grunt][7]
 - UglifyJS
 - UglifyCss
-- a database server, preferrably [MySQL][7] 5.x (we haven't tried any other DB yet...)
+- a database server, preferrably [MySQL][8] 5.x (we haven't tried any other DB yet...)
 - a web server...
 
 To install the required dependencies (apart from the database and web server) :
 
-- On OS X (using [Homebrew][8]):
+ - On OS X (using [Homebrew][9]):
```

Comment:
markdown syntax: start lists with a space

I see that I forgot to apply this to the previous list